libmachine/provision/configure_swarm.go

```diff
@@ -117,17 +118,23 @@ func configureSwarm(p Provisioner, swarmOptions swarm.Options, authOptions auth.
 			Name:              "always",
 			MaximumRetryCount: 0,
 		},
+		Binds: []string{hostBind},
```

Comment:
I see... Seems that this is out of the scope of the original feature request (supporting `--swarm-opt` for join nodes), how do you feel about making a separate proposal and PR for that?